EIPS/eip-8096.md

```diff
@@ -0,0 +1,42 @@
+---
+eip: 8096
+title: Increase Gas Cost of Point Evaluation
+description: Increases cost of point evaluation precompile
+author: Marcin Sobczak (@marcindsobczak), Kamil Chodoła (@kamilchodola), Marek Moraczyński (@MarekM25)
+discussions-to: https://ethereum-magicians.org/t/eip-8096-point-evaluation-precompile-gas-cost-increase/26867
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-12-02
+requires: 4844
+---
+
+## Abstract
+
+This EIP modifies the gas cost of `point evaluation` precompile introduced in [EIP-4844](./eip-4844.md).
+
+## Motivation
+
+Currently the `point evaluation` precompile is underpriced relative to its resource consumption. This EIP aims to address these discrepancies by adjusting the gas cost and making `point evaluation` precompile sufficiently efficient to enable potential further increases in the block gas limit.
+
+## Specification
+
+Upon activation of this EIP, the gas cost of calling the precompile at address `0x000000000000000000000000000000000000000A` will be doubled from `50_000` gas to `100_000` gas.
+
+Important notice: while increasing the gas cost of `point evaluation` precompile is necessary for enabling potential increases in the block gas limit, new cost equal `100_000` is not a final value and can be adjusted based on new pricings of other opcodes and precompiles.
```

Comment:
please add an html <-- TODO --> marker here to bot to flag it if you try to move it without addressing the TODO